grails-app/services/org/pih/warehouse/receiving/ReceiptService.groovy

```diff
@@ -143,7 +143,7 @@ class ReceiptService {
             partialReceiptItem.binLocation = binLocation
         }
         partialReceiptItem.lotNumber = shipmentItem.inventoryItem?.lotNumber
-        partialReceiptItem.expirationDate = shipmentItem.inventoryItem?.expirationDate
+        partialReceiptItem.expirationDate = shipmentItem.requisitionItem?.expirationDate ?: shipmentItem.inventoryItem?.expirationDate
```

Comment:
This is not how I would expect this to work. I think if we're storing expiration date on the requisition item it would be temporarily for that page. But the value should actually be set on the inventory item. I need to discuss with Kelsey though because we may not want to override the expiry date but instead create a new inventory item.